hpedockerplugin/file_manager.py

```diff
@@ -604,12 +621,39 @@ def mount_share(self, share_name, share, mount_id):
                 }
             }
             share['path_info'] = node_mnt_info
+            if fUser or fGroup or fMode:
+                LOG.info("Inside fUser or fGroup or fMode")
+                is_first_call = True
+                try:
+                    fUName, fGName = self._hpeplugin_driver.usr_check(fUser,
+                                                                      fGroup)
+                    if fUName is None or fGName is None:
+                        msg = ("Either user or group does not exist on 3PAR "
+                               "Please create local users and group with "
+                               "required user id and group is")
+                        LOG.error(msg)
```

Comment:
Is the last line in the error message should be `required user id and group id` ?

Reply:
Yes this line is required. I will correct the typo.
On 3PAR user name or user group can have any name but ids should match with host user/group ids.